test/node-rpc-test.js

```diff
@@ -60,4 +61,16 @@ describe('RPC', function() {
     const info = await nclient.execute('getinfo', []);
     assert.strictEqual(info.blocks, 0);
   });
+
+  it('should rpc getnetworkinfo', async () => {
+    const info = await nclient.execute('getnetworkinfo', []);
+
+    const localservicenames = [];
+    for (const [service, bit] of Object.entries(services)) {
```

Comment:
Technically this a duplicate of the same code it's testing.

Could do this instead:
```patch
diff --git a/test/node-rpc-test.js b/test/node-rpc-test.js
index d8d6fbd58..f0f9d5872 100644
--- a/test/node-rpc-test.js
+++ b/test/node-rpc-test.js
@@ -64,13 +64,6 @@ describe('RPC', function() {
 
   it('should rpc getnetworkinfo', async () => {
     const info = await nclient.execute('getnetworkinfo', []);
-
-    const localservicenames = [];
-    for (const [service, bit] of Object.entries(services)) {
-      if (node.pool.options.hasServices(bit))
-        localservicenames.push(service);
-    }
-
-    assert.deepEqual(info.localservicenames, localservicenames);
+    assert.deepEqual(info.localservicenames, ['NETWORK', 'WITNESS']);
   });
 });
```